README.md

```diff
@@ -1 +1 @@
-# provider-http
+# provider-http

```

Comment:
It would be great to add some more information to the main README here about what this provider does, how folks are expected to use it, some examples and pointers, etc. That will help people use it successfully but also make it easier for new potential contributors to get context and maybe get involved! 😊